commercial-paper/README.md

```diff
@@ -265,6 +574,33 @@ node buy.js
 java -cp target/commercial-paper-0.0.1-SNAPSHOT.jar org.digibank.Buy
 ```
 
+If you have just executed a `buy` transaction above - jump to the `redeem` transaction below - otherwise execute the _buy_/_transfer_ sequence as described earlier.
```

Comment:
The alternative needs to be presented, or at least hinted on, earlier, otherwise this not an "If" because we literally just instructed the user to execute the buy transaction and it is too late for the user to change their minds.